csharp-architectural-patterns/BrokerPattern/MessagingService/MessageService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Azure.Messaging.ServiceBus;
+
+namespace MessageService;
+
+public class MessageService : IMessageService
+{
+    Action<string>? messageHandlerCallback;
+    private static readonly string connectionString = "{service bus connection string}";
+
+    static ServiceBusClient ServiceBusClient => CreateClient();
```

Comment:
This should be `private static readonly` and the `=>` should be just `=`. As it is written now we are still creating a new client for every access of `ServiceBusClient`